jfc-food-company/package.json

```diff
@@ -11,7 +11,10 @@
     "react-dom": "^16.13.1",
     "react-router-dom": "^5.2.0",
     "react-scripts": "3.4.3",
+<<<<<<< HEAD
```

Comment:
Could  you delete this line? I think This is why netlfiy isn't passing the tests